storm-core/src/jvm/backtype/storm/serialization/DefaultSerializationDelegate.java

```diff
@@ -17,11 +17,7 @@
  */
 package backtype.storm.serialization;
```

Comment:
Done. As part of this I also deleted ThriftBridgeSerializer which depended on the now deprecated DefaultSerializationDelegate. So we now using ThriftSerializationDelegate as the default.